java-frontend/src/main/java/org/sonar/java/classpath/AbstractClasspath.java

```diff
@@ -78,14 +79,25 @@ protected AbstractClasspath(Configuration settings, FileSystem fs, InputFile.Typ
   }
 
   protected List<File> getJdkJars() {
-    return settings.get(ClasspathProperties.SONAR_JAVA_JDK_HOME)
+    List<File> jdkClassesRoots = settings.get(ClasspathProperties.SONAR_JAVA_JDK_HOME)
       .flatMap(AbstractClasspath::existingDirectoryOrLog)
       .map(File::toPath)
       .map(JavaSdkUtil::getJdkClassesRoots)
       .orElse(Collections.emptyList());
+    logResolvedFiles(ClasspathProperties.SONAR_JAVA_JDK_HOME, jdkClassesRoots);
+    return jdkClassesRoots;
+  }
+
+  static void logResolvedFiles(String property, Collection<File> files) {
+    if (LOG.isDebugEnabled() && !files.isEmpty()) {
```

Comment:
I was just wondering, Maybe it will be still useful to report empty list to say that files weren't resolved. Wdyt?